### PR TITLE
feat: add KEYS_API_URL configuration

### PR DESCRIPTION
**Pull request by eddort**

### CODE REVIEW

**Comment by Amuhar** [comment]
Lets add info about this configurations in README?

File: src/common/config/in-memory-configuration.ts [hunk 1]
```diff
@@ -128,15 +128,24 @@ export class InMemoryConfiguration implements Configuration {
...
+  KEYS_API_PORT = 0;
 
-  @IsOptional()
+  @ValidateIf((conf) => !conf.KEYS_API_URL)
+  @IsNotEmpty()
+  @IsString()
+  KEYS_API_HOST = '';
+
+  @ValidateIf((conf) => {
+    return !conf.KEYS_API_PORT && !conf.KEYS_API_HOST;
```
Comment (Amuhar) [inline src/common/config/in-memory-configuration.ts:144]: so we want to use   KEYS_API_URL only if both conf.KEYS_API_PORT  and conf.KEYS_API_HOST were not set, right ?

File: src/common/config/config-loader.service.spec.ts [hunk 2]
```diff
@@ -106,6 +125,96 @@ describe('ConfigLoaderService base spec', () => {
...
+        KEYS_API_URL,
+        KEYS_API_HOST,
+        KEYS_API_PORT,
+      });
+      const [validationErrors] = await extractError(
+        configLoaderService.loadSecrets(prepConfig),
+      );
+      toHaveProblemWithRecords(['KEYS_API_HOST'], validationErrors);
+    });
+  });
```
Comment (Amuhar) [inline src/common/config/config-loader.service.spec.ts:216]: maybe we also need test ('KEYS_API_URL, KEYS_API_HOST,  KEYS_API_PORT are set'

**Comment by Amuhar** [comment]
lgtm, good work
wrote minor comment

**Review by Amuhar — APPROVED** [review]

**Review by infloop — APPROVED** [review]

**Pull request merged**